### ISSUE TITLE
[5.4] Alt Text for Link Image

### ISSUE THREAD

**Issue by sertlan** [joomla/joomla-cms#47727]
### What happened?

Please add the "Link Image Alt Text" field to the "Link Type" tab of the menu item.

The "Link Image" field is filled in.
The "Link Image Alt Text" field is missing.
As a result, the `alt` attribute in the `<img>` tag is empty.

### Version

5.4

### Expected result

"Link Image Alt Text" field is present.

### Actual result

"Link Image Alt Text" field is missing.

### System Information

Joomla! 5.4.5

**Comment by brianteeman** [comment]
this is what i get

<img width="806" height="138" alt="Image" src="https://github.com/user-attachments/assets/64659559-7eb1-419f-a991-2582f71b4f86" />

**Comment by sertlan** [comment]
> this automated test is wrong

@brianteeman This opinion is incorrect because errors and warnings in these tests negatively impact sites' rankings in Google and Bing search results.

**Comment by sertlan** [comment]
However, if the link image fails to load, there will be an empty space containing an invisible link. This means the image is functional rather than decorative.
